docs/content/learn/tutorials/control/basic-profile.md

```diff
@@ -0,0 +1,1282 @@
+---
+title: Creating a Profile
+description: A tutorial on creating a Baseline using an OSCAL Profile.
+weight: 20
+suppresstopiclist: true
+toc:
+  enabled: true
+aliases:
+  - /tutorials/profile/
+  - /learn/tutorials/profile/
+---
+
+This tutorial covers creating and modifying a subset of controls from a catalog in OSCAL by using the OSCAL profile model. Such a subset of controls is often referred to as a "control baseline".
+
+Before reading this tutorial you should:
+- Have some familiarity with the [XML](https://www.w3.org/standards/xml/core), [JSON](https://www.json.org/), or [YAML](https://yaml.org/spec/) formats.
+- Review the OSCAL control layer [overview](/concepts/layer/control/).
+- Review the OSCAL [catalog](/concepts/layer/control/catalog/) and [profile](/concepts/layer/control/profile/) model overview pages.
+- Review the tutorial on [Creating a Control Catalog](/learn/tutorials/catalog/).
+
+## What is an OSCAL Profile?
+
+An OSCAL profile is a specific set of security controls selected and modified when needed from one or more control catalogs for use in managing risks in an information system. Such a profile is also known as [baseline][baseline-definition], or overlay in the NIST SP 800-37 rev2, [Risk Management Framework for Information Systems and Organizations: A System Life Cycle Approach for Security and Privacy](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-37r2.pdf).
+
+An OSCAL profile is a machine-readable representation of a baseline, expressed using the OSCAL [profile model][profile-docs], which includes contextualizing documentation and metadata.
+In the most basic sense, an OSCAL profile is a collection of "pointers" to other catalog(s)'s controls, along with instructions to tailor the controls and change how the controls are grouped.
+
+An OSCAL profile can be transformed into an OSCAL catalog through a process named *profile resolution*, which is described in the [Profile Resolution Specification](/concepts/processing/profile-resolution/).
+The output *resolved* catalog contains the controls selected, tailored, and grouped by the profile.
```

Comment:
Minor non-blocking: small change of conjunction and make wording a little more accurate.

```suggestion
The output *resolved* catalog contains the controls selected, tailored, or (optionally) grouped by the profile.
```